ansible/files/configuration/configuration.json

```diff
@@ -23,6 +23,7 @@
   }, {
     "Classification": "spark-defaults",
     "Properties": {
+      "spark.metrics.namespace": "spark",
```

Comment:
We set namespace to "spark" so that Spark doesn't use the default namespace of applicationId, which is unique for every cluster.

By setting this to spark and not configuring a prefix for the StatsdSink, we end up with metric paths like:

```
  spark.driver.LiveListenerBus.queue.appStatus.listenerProcessingTime.mean_rate
  spark.1.executor.shuffleTotalBytesRead
```